linkstatus/linkstatus.py

```diff
@@ -20,11 +20,11 @@ def link_status(link, timeout=5):
     Returns:
         tuple of status (bool) and status code
     """
-
+    headers = {"User-Agent": "linkstatus/0.8"}
```

Comment:
what you think about fetching current version than hardcoding? like
```
>>> import pkg_resources
>>> pkg_resources.get_distribution("linkstatus").version
'0.8.dev3+g0f6b73b.d20200329'
```

Reply:
Done